pkgs/tools/system/gotop/default.nix

```diff
@@ -2,15 +2,15 @@
 
 buildGoPackage rec {
   name = "gotop-${version}";
-  version = "1.7.1";
+  version = "2.0.1";
 
   goPackagePath = "github.com/cjbassi/gotop";
 
   src = fetchFromGitHub {
     repo = "gotop";
     owner = "cjbassi";
     rev = version;
-    sha256 = "0dxnhal10kv6ypsg6mildzpz6vi1iw996q47f4rv8hvfyrffhzc9";
+    sha256 = "0xpm8nrn53kz65f93czflgdgr2a33qfi1w0gsgngrmaliq1vlpji";
   };
 
   goDeps = ./deps.nix;
```

Comment:
It looks like the deps are vendored in the github release tarball, can we drop deps.nix while we're at it and git rid of this line? :)

Reply:
@srhb nice one!